potassium/potassium.py

```diff
@@ -105,11 +105,11 @@ def task(func, req):
         return flask_app
 
     # serve runs the http server
-    def serve(self, port = 8000):
+    def serve(self, host="0.0.0.0", port = 8000):
```

Comment:
technically breaking change if people did positional use of port, but v0 and only recently launched so assuming zero affected